PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -48,7 +48,11 @@
         cms.InputTag("pfCombinedMVAV2BJetTags"),
         # CTagging
         cms.InputTag('pfCombinedCvsLJetTags'),
-        cms.InputTag('pfCombinedCvsBJetTags')
+        cms.InputTag('pfNegativeCombinedCvsLJetTags'),
+        cms.InputTag('pfPositiveCombinedCvsLJetTags'),
+        cms.InputTag('pfCombinedCvsBJetTags'),
+        cms.InputTag('pfNegativeCombinedCvsBJetTags'),
+        cms.InputTag('pfPositiveCombinedCvsBJetTags')
```

Comment:
Did you try removing these? It should have no effect in the BTagAnalyzer, but in this way they do not get processed when running the default PAT sequence. This will allow us to submit this PR to CMSSW without getting too much in trouble (we do not change the event content)